stable/insights-agent/values.yaml

```
@@ -575,3 +575,15 @@ falcosecurity:
         minimumpriority: ""
         mutualtls: false
         checkcert: true
+
+kyverno:
+  enabled: false
+  schedule: "rand * * * *"
+  image:
+    repository: quay.io/fairwinds/kyverno
+    tag: "0.1.1"
```

I'd suggest unpinning the patch version, like we do for other plugins
```suggestion
    tag: "0.1"
```